fleetspeak.md

```diff
@@ -86,10 +86,16 @@ packages:
     (venv) $ pip install -e grr/server/[mysqldatastore]
     (venv) $ deactivate
 
+If you encounter any problems related to `mysqlclient` during the installation of `grr/server` when using MariaDB, then try the following:
+
+    (venv) $ pip install mysqlclient
+    (venv) $ pip install -e grr/server/
+    (venv) $ deactivate
+
 Configuration
 -------------
 
-### MySQL
+### MySQL (MariaDB)
```

Comment:
I would use "MySQL / MariaDB", the current form to me means "MySQL, but MariaDB in reality".

Reply:
Oh, sorry, you're right, I didn't read it like this in my mind. Changed it. Thanks!